PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -147,6 +147,7 @@ private void sendPayPalRequest(
             final PayPalNativeRequest payPalRequest,
             final Configuration configuration
     ) {
+        braintreeClient.setPayPalContextID(configuration.getPayPalClientId());
```

Comment:
I think we want to pass the order ID here instead of the client ID

Reply:
Good catch!!! Working on it!!

Reply:
Addessed on: 02effcceb32ad44fc7e31fa73f51cd704c0e17cc